github-dispatches/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.2
+
+LABEL version="1.0.0"
+LABEL repository="http://github.com/redhat-cop/github-actions"
+LABEL homepage="http://github.com/redhat-cop/github-actions/github-dispatches"
+LABEL maintainer="Red Hat CoP"
+LABEL "com.github.actions.name"="github-dispatches"
+LABEL "com.github.actions.description"="Triggers a GitHub CI dispatches event."
+LABEL "com.github.actions.branding.icon"="monitor"
+LABEL "com.github.actions.branding.color"="purple"
+
+RUN microdnf install --assumeyes --nodocs curl wget && \
+    microdnf clean all && \
+    curl --version && \
+    wget --version
+
+RUN export JQ_VERSION=1.6 && \
+    wget --no-verbose --directory-prefix=/tmp https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 && \
+    chmod +x /tmp/jq-linux64 && \
+    ln -s /tmp/jq-linux64 /usr/local/bin/jq && \
+    jq --version
```

Comment:
I think this is the only place we're using `wget`, if so I'd suggest to replace it with curl and skip installing wget altogether.
```
RUN export JQ_VERSION=1.6 && \
    curl -Lqo /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 && \
    chmod +x /usr/local/bin/jq && \
    jq --version
```